### PR TITLE
May Heartbeat less main image

### PR DESCRIPTION
**Pull request by jendefig**
Here is the May Heartbeat ready for review.  

🕚 It should go out today, but I don't have an updated graphic from the graphics guy I've been working with.  He is celebrating Eid.  I've given him feedback and more to work with so that the next pass should be on point. 🤞🏼 He said he needs another day.

### CODE REVIEW

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+and is excited to work on open source software. In her spare time you may find
+her dancing the tango! 💃🏻 Welcome Sveta!
+
+Exemplifying our diverse team in one fell swoop,
+[Casper da Costa-Luis](https://www.cdcl.ml) was born in Kenya, has an Indian
+Passport and lives in the UK. He has been working on DVC for a couple of years
+and contributing to open source since 2014. He now joins us full time on the CML
+team after completing his PhD in Medical Imaging. Fun facts about Casper include
+his becoming the U18 chess champion of Kenya when he was 14 and that he is a
+qualified SCUBA diver. 🤿
```
Comment (casperdcl) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:54]: TMI

```suggestion
[Casper da Costa-Luis](https://www.cdcl.ml) has lived in three continents. He
has been working on DVC for a couple of years and is a long-standing contributor
to open source. He now joins us on the CML & Docs teams after completing his PhD
in Medical Imaging. Fun facts about Casper include his becoming the U18 chess
champion of Kenya when he was 14 and being a qualified SCUBA diver. 🤿
```

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+[Emre Sahin](https://github.com/iesahin) joins us on the DevRel team as a
+techinical writer/ML enthusiast/AI dreamer/tutorial builder from Instanbul,
+Turkey. A self-described zelaot for technologies, Emre has worked in many
+development/ML related projects and has been programming in Python since v. 1.7.
+We are excited for Emre to bring you excellent technical content! ✍🏼
+
+[Tapa Dipti Situala](https://www.linkedin.com/in/tapa-dipti-sitaula/) joins us
```
Comment (casperdcl) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:62]: corerct teh spleling

```suggestion
[Tapa Dipti Sitaula](https://www.linkedin.com/in/tapa-dipti-sitaula/) joins us
```

**Review by casperdcl — CHANGES_REQUESTED** [review]
probably should ping @tapadipti @iesahin @ssachkovskaya to check for silly mistakes

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+Exemplifying our diverse team in one fell swoop,
+[Casper da Costa-Luis](https://www.cdcl.ml) has lived in three continents. He
+has been working on DVC for a couple of years and is a long-standing contributor
+to open source. He now joins us on the CML & Docs teams after completing his PhD
+in Medical Imaging. Fun facts about Casper include his becoming the U18 chess
+champion of Kenya when he was 14 and being a qualified SCUBA diver. 🤿
+
+[Emre Sahin](https://github.com/iesahin) joins us on the DevRel team as a
+techinical writer/ML enthusiast/AI dreamer/tutorial builder from Instanbul,
```
Comment (skshetry) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:56]:
```suggestion
technical writer/ML enthusiast/AI dreamer/tutorial builder from Instanbul,
```

Comment (casperdcl) [reply content/blog/2021-05-15-may-21-dvc-heartbeat.md:56]: :smile: 
```suggestion
technical writer/ML enthusiast/AI dreamer/tutorial builder from Istanbul,
```

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+Turkey. A self-described zelaot for technologies, Emre has worked in many
```
Comment (casperdcl) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:57]:
```suggestion
Turkey. A self-described zealot for technologies, Emre has worked in many
```

Comment (iesahin) [reply content/blog/2021-05-15-may-21-dvc-heartbeat.md:57]: Thank you for fixing my things :)) @casperdcl @skshetry

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+  opportunities, tutorials with integrations, conference videos, Discord 
+  server growth, and more!
+descriptionLong: |
+  This month you will find:
+  🚀 info on our growing team 
+  📖 MLOps learning opportunities,
+  🔃 tutorials with integrations, 
+  🎥 conference videos from our team members, 
+  💥 Discord server growth, and 
+  more!
```
Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:15]: 💅 `more` - move it to the previous line?

**Comment by jendefig** [comment]
Thanks @casperdcl, @skshetry for the corrections!!!

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+her dancing the tango! 💃🏻 Welcome Sveta!
+
+Exemplifying our diverse team in one fell swoop,
+[Casper da Costa-Luis](https://www.cdcl.ml) has lived in three continents. He
+has been working on DVC for a couple of years and is a long-standing contributor
+to open source. He now joins us on the CML & Docs teams after completing his PhD
+in Medical Imaging. Fun facts about Casper include his becoming the U18 chess
+champion of Kenya when he was 14 and being a qualified SCUBA diver. 🤿
+
+[Emre Sahin](https://github.com/iesahin) joins us on the DevRel team as a
```
Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:55]: use the right Ş -> Şahin

Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:55]: DevRel -> DVC team (for now at least)

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+with you Tapa! 🚀
+
+## Open Positions
+
+And we're still hiring!
+
+[**Check out our three open roles**](https://weworkremotely.com/company/iterative)
+for:
+
+- [**Senior Frontend Engineer**](https://weworkremotely.com/remote-jobs/iterative-senior-frontend-engineer)
```
Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:75]: link is broken, should be `front-end` (should we fix previous posts), please check other links

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+[Head to the link](https://github.com/sscardapane/reprodl2021) to star the repo
+and cheer him on. We will be! 🙌🏼
+
+You see what I did there, right? **Reproducible**... **Deep Learning**.....  
+Get it? Layers of wit people. I learned from the best! Just wanted to make sure
+you were paying attention!
+
+https://media.giphy.com/media/6ra84Uso2hoir3YCgb/giphy.gif
+
+# From the Community
```
Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:161]: Should we make it before Team Talks, before Other ML Ops ... ?

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+
+And now our beloved community! It's been another month full of community
+goodness and we are grateful!
+
+![Thank you](https://media.giphy.com/media/jmqWAjoxFCxJNHD2Kz/giphy.gif)
+
+### Curvenote with DVC tutorials
```
Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:168]: Same - feels like we should be prioritizing community talks vs us?

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 10]
```diff
@@ -0,0 +1,239 @@
...
+Don't miss our
+[Meetup](https://www.meetup.com/DVC-Community-Virtual-Meetups/events/277245660)
+May 27th at 3:00pm UTC, where we will hear from Puneeth Pai as mentioned above
+👆🏽, as well as another user putting DVC and CML into action on his team, and
+finally from David Ortega discussing CML pull requests! Bring your questions!
+We're here to help!
+
+## Tweet Love ❤️
+
+https://twitter.com/FullStackML/status/1389701264997445635?s=20
```
Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:228]: Do we have some community Tweets from the last month? :)

**Comment by shcheklein** [comment]
@jendefig please merge master, build is broken right now (it's fixed in master).

**Review by shcheklein — CHANGES_REQUESTED** [review]
Great stuff as usual 😎 

Let's fix some minor issues and fix the build to review it properly - ping me when it's deployed. Thanks 🙏

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
+﻿---
+title: May '21 Heartbeat
+date: 2021-05-15
+description: |
+  Monthly updates are here! We've hit 30 team members! MLOps learning 
+  opportunities, tutorials with integrations, conference videos, Discord 
+  server growth, and more!
+descriptionLong: |
+  This month you will find:
+  🚀 info on our growing team
```
Comment (tapadipti) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:10]: I think content on the team growth can be included after the other sections (in this list as well as in the details below) to highlight the other sections more.

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md (hunk 10 above)
Comment (jendefig) [reply content/blog/2021-05-15-may-21-dvc-heartbeat.md:228]: Inserting Paige's Tweet.  Guess there are pluses to the delay...

File: content/blog/2021-05-15-may-21-dvc-heartbeat.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
+﻿---
+title: May '21 Heartbeat
+date: 2021-05-15
```
Comment (shcheklein) [inline content/blog/2021-05-15-may-21-dvc-heartbeat.md:3]: do we need to update dates?

**Comment by shcheklein** [comment]
I see that it's still failing to deploy - have you tried to run it locally? Could you please merge master?

**Comment by shcheklein** [comment]
@jendefig you still need to merge master. Build is failing still :(

**Comment by jendefig** [comment]
> @jendefig you still need to merge master. Build is failing still :(

 Ok.  Changed the dates again and merged master.  I was having and issue with viewing in the local host.  Had an error, so not sure all is working.

**Comment by shcheklein** [comment]
Hey, @jendefig let me check.

I've merged the master back, let's see if deploys now. That's what I've done so far:

```bash
gh pr checkout 2477 # this is not needed in your case, that was for me to take this PR
git checkout master
git pull
git checkout -  # this is to get back to this branch
git merge master
git push origin HEAD
```

**Comment by shcheklein** [comment]
@jendefig okay, it has been deployed now. ... let me do the final round of checks ...

**Comment by shcheklein** [comment]
Did some image optimizations:

Screen Shot 2021-05-21 at 4 31 49 PM

**Comment by shcheklein** [comment]
Waiting for the deploy and checks here to pass again and to review ...

**Comment by shcheklein** [comment]
Okay, it's good to go now 🚀

**Review by shcheklein — APPROVED** [review]

**Comment by jendefig** [comment]
> Did some image optimizations:
> 
> Screen Shot 2021-05-21 at 4 31 49 PM

FYI.  I did run all the images through this once.

**Comment by shcheklein** [comment]
@jendefig yep, hard to tell what happened. Probably forgot to `git push`?

**Pull request merged**